### PR TITLE
session module type added and removed (non-) static methods from session

### PR DESCRIPTION
**Pull request by mataron**
case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [ ] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [ ] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by rhysd** [comment]
Indentation looks wrong.  Please format the change.

**Comment by rhysd** [comment]
And please add tests for your changes.

**Comment by mataron** [comment]
well, as I made apparent, this is the first time testing typings :)

**Comment by rhysd** [comment]
:+1:

But I forgot notifying you:  There're many open PRs currently (please search this repository's open PRs with 'github-electron:').  So you may need to resolve conflicts later.

**Comment by mataron** [comment]
np, will do

**Comment by vvakame** [comment]
@mataron #8723 merged!

**Comment by miniak** [comment]
After merging https://github.com/DefinitelyTyped/DefinitelyTyped/pull/8732, I think all of your changes will already be there.

**Comment by vvakame** [comment]
@mataron please resolve conflict.

**Comment by mhegazy** [comment]
@mataron please update the PR.

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -1912,6 +1941,281 @@ declare namespace Electron {
 		remote: Electron.Remote;
 		webFrame: Electron.WebFrame;
 	}
+
+	interface HTMLWebviewElement extends HTMLElement {
```
Comment (miniak) [inline github-electron/github-electron.d.ts]: This is duplicate

**Comment by miniak** [comment]
This PR is obsolete and should be closed

**Comment by mhegazy** [comment]
Thanks for the reply. closing.

**Pull request closed without merging**